### ISSUE TITLE
why it still has non trainable params? even tho I set it to .trainable = True?

### ISSUE THREAD

**Issue by caesariodito** [caesariodito/k-thesis#5]
![image](https://github.com/caesariodito/k-thesis/assets/75000641/641c03aa-9e67-4ecb-a5b9-101b00b64e92)

I've heard that batch normalization doesn't update when I use ".trainable = True". I believe I read it on TensorFlow.

Might consider looping through the loaded Xception model and see what layers have false values on trainable.

**Comment by caesariodito** [comment]
closing this because the project is finished
(I've graduated and finished my thesis)
